### PR TITLE
Added notice styling and traslatable string in default templates for CMS home and no-route

### PR DESCRIPTION
**Pull request by dev-jq**
### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by Den4ik** [comment]
@jq91 Please sign CLA

**Comment by jq91** [comment]
@Den4ik CLA already signed.

**Comment by Den4ik** [comment]
@magento run all tests

File: app/code/Magento/Cms/view/frontend/templates/default/home.phtml [hunk 1]
```diff
@@ -4,4 +4,6 @@
  * See COPYING.txt for license details.
  */
 ?>
-There was no Home CMS page configured or found.
+<div class="message notice">
+    <div><?= __('There was no Home CMS page configured or found.'); ?></div>
```
Comment (Den4ik) [inline app/code/Magento/Cms/view/frontend/templates/default/home.phtml]: Output should be escaped with by `$escaper->escapeHtml()`

File: app/code/Magento/Cms/view/frontend/templates/default/no-route.phtml [hunk 2]
```diff
@@ -4,4 +4,6 @@
  * See COPYING.txt for license details.
  */
 ?>
-There was no 404 CMS page configured or found.
+<div class="message notice">
+    <div><?= __('There was no 404 CMS page configured or found.'); ?></div>
```
Comment (Den4ik) [inline app/code/Magento/Cms/view/frontend/templates/default/no-route.phtml]: Output should be escaped with by `$escaper->escapeHtml()`

**Review by Den4ik — CHANGES_REQUESTED** [review]
@jq91 Please look at my comment

**Comment by jq91** [comment]
@Den4ik escapeHtml() has been added
https://github.com/magento/magento2/pull/36126/commits/982e16f520c7581cb8fb71022c0a503a09918e36

**Comment by Den4ik** [comment]
@magento run all tests

**Comment by Den4ik** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Functional Tests CE, Sample Data Tests B2B, Unit Tests

**Comment by jq91** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Functional Tests CE

**Pull request closed without merging**